### DISCUSSION TITLE
Get balance on address

### DISCUSSION THREAD

**Discussion by skeutmeier** [MrNaif2018/bitcart#346] [Q&A]
Hi,
I am building a client and I'm using the swagger API.

I recognized that if there is apayment made to an address after the timeout period (or if invoice is markde as paid), the new TX doesn't show up in the invoice.

Is there a way to get the balance of an address via the swagger api?

**Comment by MrNaif2018** [comment]
No, I think no, only wallet balance. In general we don't track invoices after expiry time or it has been paid
